stdlib/2and3/xml/etree/ElementTree.pyi

```diff
@@ -70,6 +70,21 @@ class Element(MutableSequence['Element']):
     def remove(self, subelement: 'Element') -> None: ...
     def set(self, key: _str_argument_type, value: _str_argument_type) -> None: ...
     def __bool__(self) -> bool: ...
+    @overload
+    def __delitem__(self, i: int) -> None: ...
```

Comment:
Does this need an overload? Looks like you can just use a union.

Reply:
Done.